### PR TITLE
Update Facebook profile fields

### PR DESCRIPTION
**Pull request by loay**
connect to https://github.com/strongloop/loopback-component-passport/issues/147

### CODE REVIEW

**Comment by loay** [comment]
@superkhau @jannyHou PTAL. Thanks.

File: README.md [hunk 1]
```diff
@@ -87,18 +87,16 @@ $ npm install
 ```
 ### 4. Facebook profile info
 
-If you require additional information from a Facebook profile such as a name or a gender, you can obtain it by updating `node_modules\passport-facebook\lib\strategy.js`and replacing:
+In a recent update, Facebook no longer returns all the fields by default such as email, gender, timezone ..etc.
```
Comment (superkhau) [inline README.md]: all fields by default (email, gender, timezone, etc).

File: README.md [hunk 2]
```diff
@@ -87,18 +87,16 @@ $ npm install
...
+If you require some additional information, you can update the providers template to reflect these changes.
```
Comment (superkhau) [inline README.md]: If you need more information, modify the providers template.

File: README.md [hunk 3]
```diff
@@ -87,18 +87,16 @@ $ npm install
...
 
+The current template is updated with the following:
```
Comment (superkhau) [inline README.md]: The current template contains:

File: README.md [hunk 4]
```diff
@@ -87,18 +87,16 @@ $ npm install
...
-```
-
-with
+"profileFields": ["gender", "link", "locale", "name", "timezone", "verified", "email", "updated_time"],
 
 ```
-this._profileURL = options.profileURL ||
-    'https://graph.facebook.com/v2.2/me?fields=first_name,gender,last_name,link,locale,name,timezone,verified,email,updated_time';
-```
+You can add/remove the fields according to your application needs. For more info about the
```
Comment (superkhau) [inline README.md]: We recommend modifying the fields to suit your needs. For more information regarding the providers template, see http://loopback.io/doc/en/lb2/Configuring-providers.json.html.

**Comment by superkhau** [comment]
@loay Please make the above grammatical changes, otherwise LGTM.

**Comment by jannyHou** [comment]
LGTM, new explanation is clearer to me 👍

**Pull request merged**